librz/core/cmd_analysis.c

```diff
@@ -5196,6 +5197,7 @@ static void cmd_analysis_esil(RzCore *core, const char *input) {
 		}
 		break;
 	case 'b': // "aeb"
+		// BUG : aeb causes SEGMENT FAULT
```

Comment:
Was resolved, can be removed now. But I recommend to add a test for this, so we could check both ESIL and RZIL implementations.